rest/route/project.go

```diff
@@ -1048,3 +1048,59 @@ func (h *projectParametersGetHandler) Run(ctx context.Context) gimlet.Responder
 
 	return gimlet.NewJSONResponse(res)
 }
+
+////////////////////////////////////////////////////////////////////////
+//
+// PUT /rest/v2/projects/value/{toReplace}/{replacement}
+
+type rotation struct {
+	toReplace   string
+	replacement string
+}
+
+type projectVarsPatchHandler struct {
+	replaceVars *rotation
+	user        *user.DBUser
+	args        []byte
+
+	sc       data.Connector
+	settings *evergreen.Settings
+}
+
+func makeProjectVarsPatch(sc data.Connector, settings *evergreen.Settings) gimlet.RouteHandler {
```

Comment:
Since this ended up being a Put, can we please change the name to makeProjectVarsPut?